src/app/webbrowser/TabComponent.qml

```diff
@@ -279,5 +296,15 @@ Component {
                 }
             }
         }
+        function toggleReaderMode() {
+            if (browserTab.readerMode) {
+                browserTab.reload()
```

Comment:
i thought about storing the entire dom in the `makeDomReadable` function, but I'm not sure it's worth the tradeoff.
I don't really understand how that'd work with something like a video that's playing (if a reference is stored in a "shadow dom", does it keep playing? etc)